multimodal/src/autogluon/multimodal/data/preprocess_dataframe.py

```diff
@@ -303,7 +317,7 @@ def fit(self, X: Optional[pd.DataFrame] = None, y: Optional[pd.Series] = None):
     def transform_text(
         self,
         df: pd.DataFrame,
-    ) -> Dict[str, List[str]]:
+    ) -> Tuple[Dict[str, List[str]], Dict[str, str]]:
```

Comment:
We may want to update the description of the returns accordingly (in all transform_xxx)

Reply:
Good suggestion. Updated the return docstrings.